src/Conference/ConferenceDetails.jsx

```diff
@@ -48,17 +55,26 @@ const ConferenceDetails = ({ conference, id, isLoading, error }) => {
     <StyledForm>
       <StyledCard>
         <StyledCardHeading className="name">{name}</StyledCardHeading>
-        <StyledLink to={`/${id}/edit`}>Edit Conference</StyledLink>
+        {allCookies.sessionToken && (
```

Comment:
Optimally you would look at abstracting this away from the cookie directly and into a util or provider of your own to make this more maintainable.

Reply:
We will be looking at implementing cookies in a cleaner way, as part of the solution addressed in the other comment